alfasim_sdk/constants.py

```diff
@@ -42,3 +44,50 @@
 """
 The dict key which identifies the required version of alfasim-sdk for a given plugin
 """
+
+
+class HydrodynamicModelType(Enum):
```

Comment:
@arthursoprana, @ggrbill , @sofiapbittencourt , @edersongrein 

Any suggestions for the docstring of the following items?
- FiveFieldsSolid
- FiveFieldsWater
- FiveFieldsCO2
- ThreeLayersNoBubbleGasOilWater
- ThreeLayersWaterWithCO2

Just remembering that this content of this docstring goes directly to the ALFAsim-SDK documentation.

Reply:
I suggest to remove just from the docstrings and put some comment in the code saying that those types are not supported (since they are not available on GUI), or list them at the end of docstring also saying that they are not supported.

Reply:
As @nicoddemus suggested in Rocket.Chat, add a `Internal` or `Do not use` tag.